elastic/kibana
# Test SubAction schemas

Resolves: #177025

I figured out a way to get the SubActions schemas and used it rather than the suggested way in the issue.

## To verify:

Change one of the SubAction's schema and run the integration test, It should fail.
